products/alibaba-ack.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Alibaba ACK
+category: service
+tags: alibaba managed-kubernetes
+iconSlug: alibabacloud
+permalink: /alibaba-ack
+alternate_urls:
+-   /ack
+-   /alibaba-container-service-for-kubernetes
+versionCommand: kubectl version
+releasePolicyLink: https://www.alibabacloud.com/help/en/ack/ack-managed-and-ack-dedicated/user-guide/support-for-kubernetes-versions
+releaseDateColumn: true
+eolColumn: End of Support
+
+releases:
+-   releaseCycle: "1.32"
+    releaseDate: 2025-01-01
+    eol: 2026-01-31
+    latest: '1.32.1-aliyun.1'
+
+-   releaseCycle: "1.31"
+    releaseDate: 2024-09-01
+    eol: 2025-09-30
+    latest: '1.31.1-aliyun.1'
+
+-   releaseCycle: "1.30"
+    releaseDate: 2024-06-01
+    eol: 2026-06-30
+    latest: '1.30.7-aliyun.1'
+
+-   releaseCycle: "1.28"
+    releaseDate: 2023-10-01
+    eol: 2025-10-31
+    latest: '1.28.15-aliyun.1'
+
+-   releaseCycle: "1.26"
+    releaseDate: 2023-04-01
+    eol: 2025-04-30
+    latest: '1.26.15-aliyun.1'
+
+-   releaseCycle: "1.24"
+    releaseDate: 2022-09-01
+    eol: 2024-09-30
+    latest: '1.24.6-aliyun.1'
+
+-   releaseCycle: "1.22"
+    releaseDate: 2021-12-01
+    eol: 2023-10-31
+    latest: '1.22.15-aliyun.1'
+
+-   releaseCycle: "1.20"
+    releaseDate: 2021-04-01
+    eol: 2023-04-30
+    latest: '1.20.11'
+
+-   releaseCycle: "1.18"
+    releaseDate: 2020-09-01
+    eol: 2022-09-30
+    latest: "1.18.8"
+
+-   releaseCycle: "1.16"
+    releaseDate: 2020-02-01
+    eol: 2022-06-30
+    latest: "1.16.9"
+
+---
+
+> [Container Service for Kubernetes (ACK)](https://www.alibabacloud.com/en/product/kubernetes) is one of the first services to pass the Certified Kubernetes Conformance Program. 
+> ACK provides high-performance management services for containerized applications. You can manage enterprise-level containerized 
+> applications throughout the application lifecycle. This service allows you to run containerized applications in the cloud in an efficient manner.
```

Comment:
```suggestion
> [Alibaba Container Service for Kubernetes (ACK)](https://www.alibabacloud.com/en/product/kubernetes) is a managed service
> that can be used to run Kubernetes on Alibaba without needing to install, operate, and maintain your
> own Kubernetes control plane or nodes. ACK runs patched Kubernetes versions and is certified Kubernetes
> conformant for a predictable experience.
```

Suggestion to make the description more neutral (inspired by https://github.com/endoflife-date/endoflife.date/blob/master/products/eks.md).